src/pages/index.js

```diff
@@ -32,19 +32,23 @@ function createGoHireJobs() {
 }
 
 const IndexPage = () => {
+
+  useEffect(() => {
```

Comment:
Can we add the second param for the `useEfect`? Second param helps us prevent recall this function on every re-render. Just empty array `[]`